incubator/jaeger/templates/cassandra-schema-job.yaml

```diff
@@ -55,6 +59,37 @@ spec:
       {{- end }}
         resources:
 {{ toYaml .Values.schema.resources | indent 10 }}
+        volumeMounts:
+        {{- range .Values.schema.extraConfigmapMounts }}
+          - name: {{ .name }}
+            mountPath: {{ .mountPath }}
+            subPath: {{ .subPath }}
+            readOnly: {{ .readOnly }}
+        {{- end }}
+        {{- if .Values.storage.cassandra.tls.enabled }}
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/root/.cassandra/ca_cert.pem"
+            subPath: "ca_cert.pem"
+            readOnly: true
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/root/.cassandra/client_cert.pem"
+            subPath: "client_cert.pem"
+            readOnly: true
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/root/.cassandra/client_key.pem"
+            subPath: "client_key.pem"
+            readOnly: true
+          - name: {{ .Values.storage.cassandra.tls.secretName }}
+            mountPath: "/root/.cassandra/cqlshrc"
+            subPath: "cqlrhsc"
```

Comment:
not sure about this one. the cqlrhsc file should not need to come from the tls secret

Reply:
Where should it come from?

> # Creating schema in a cassandra cluster requiring client TLS certificates.
> #
> # Create a volume for the schema docker container containing four files:
> # cqlshrc: this file
> # ca-cert: the cert authority for your keys
> # client-key: the keyfile for your client
> # client-cert: the cert file matching client-key
> #
> # if there is any sort of DNS mismatch and you want to ignore server validation
> # issues, then uncomment validate = false below.
> #
> # When running the container, map this volume to /root/.cassandra and set the
> # environment variable CQLSH_SSL=--ssl
> [ssl]
> certfile = ~/.cassandra/ca-cert
> userkey = ~/.cassandra/client-key
> usercert = ~/.cassandra/client-cert
> # validate = false
>

Reply:
@naseemkullah